src/shared/components/navbar/LogoNavBar.tsx

```
@@ -0,0 +1,47 @@
+  buttonType?: ButtonType;
+}
+
+const LogoNavBar = ({ buttonType = null, ...props }: LogoNavBarProps) => {
+  const navigate = useNavigate();
+
+  console.log('현재 버튼 타입:', buttonType);
+
+  const handleRenderBtn = () => {
+    switch (buttonType) {
+      case 'login':
+        return <button onClick={() => navigate(ROUTES.LOGIN)}>로그인</button>;
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)